src/content/en/fundamentals/media/video-preload.md

```diff
@@ -0,0 +1,393 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Preload all video for faster playback.
+
+{# wf_published_on: 2017-06-21 #}
+{# wf_updated_on: 2017-06-21 #}
+
+# Video Preload = Faster Playback {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque suscipit nibh nec feugiat ornare. Nam tristique, magna non pharetra mollis, diam elit faucibus nibh, ut condimentum diam metus vel dolor. Integer facilisis magna at ex bibendum luctus. In eu tincidunt ipsum. Duis scelerisque, tortor quis condimentum finibus, est neque mollis leo, at imperdiet justo elit dapibus nisl. Integer gravida dapibus turpis, vel scelerisque dolor scelerisque eget. Sed dolor lectus, tempor vulputate bibendum eget, accumsan lobortis ex.
+
+Unless specified otherwise, this article also applies to the audio element.
+
+## Video preload attribute
+
+If the video source is an unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. In other words, [Media Source
+Extensions (MSE)] is not compatible with `preload`.
```

Comment:
@mounirlamouri can confirm but I believe `preload` could work as a fallback for MSE.